data/mods/Magiclysm/Spells/attunements/Illusionist.json

```diff
@@ -24,5 +24,84 @@
     "max_level": 35,
     "base_casting_time": 180,
     "energy_source": "MANA"
+  },
+  {
+    "id": "illusionist_hologram",
+    "type": "SPELL",
+    "name": "Battle Holograms",
+    "description": "Created to disrupt your enemies, they look exactly like you, and act like a real person.",
+    "valid_targets": [ "ground" ],
+    "effect": "summon",
+    "effect_str": "mon_battle_hologram",
+    "shape": "blast",
+    "flags": [ "SILENT", "RANDOM_TARGET", "NO_EXPLOSION_SFX", "MUST_HAVE_CLASS_TO_LEARN" ],
+    "spell_class": "ILLUSIONIST",
+    "energy_source": "MANA",
+    "min_range": 1,
+    "max_range": 1,
+    "min_damage": 5,
+    "max_damage": 55,
+    "damage_increment": 2,
+    "min_aoe": 9,
+    "max_aoe": 34,
+    "aoe_increment": 1,
+    "difficulty": 7,
+    "base_casting_time": 400,
+    "final_casting_time": 100,
+    "casting_time_increment": -10,
+    "base_energy_cost": 400,
+    "final_energy_cost": 200,
+    "energy_increment": -6,
+    "max_level": 35,
+    "min_duration": 3500,
+    "max_duration": 126000,
+    "duration_increment": 3500
+  },
+  {
```

Comment:
Should this monster go in a monster file?

Reply:
They exist only in this attunements, so i see no reason to split it